Gruntfile.js

```diff
@@ -522,4 +523,4 @@ module.exports = function (grunt) {
 
     // Default task.
     grunt.registerTask('default', ['test']);
-};
+};
```

Comment:
Here's an example of what we discussed in class about not having an end-of-file line break.  Please revert this change back.